index.html

```diff
@@ -67,11 +66,167 @@ <h3>
+          </div>
+        </div>
+        <div class="work-2">
+          <div class="img-holder"></div>
+          <div class="posts">
+            <h2>Multi-Post Stories Gain+Glory</h2>
+            <div class="flex-post">
+              <ul>
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Javascript</a></li>
+                <li><a href="#">HTML</a></li>
+              </ul>
+            </div>
+            <button type="button" class="works-button">See Project</button>
+          </div>
+        </div>
+        <div class="work-3">
+          <div class="img-holder"></div>
+          <div class="posts">
+            <h2>Multi-Post Stories Gain+Glory</h2>
+            <div class="flex-post">
+              <ul>
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Javascript</a></li>
+                <li><a href="#">HTML</a></li>
+              </ul>
+            </div>
+            <button type="button" class="works-button">See Project</button>
+          </div>
+        </div>
+        <div class="work-4">
+          <div class="img-holder"></div>
+          <div class="posts">
+            <h2>Multi-Post Stories Gain+Glory</h2>
+            <div class="flex-post">
+              <ul>
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Javascript</a></li>
+                <li><a href="#">HTML</a></li>
+              </ul>
+            </div>
+            <button type="button" class="works-button">See Project</button>
+          </div>
+        </div>
+        <div class="work-5">
+          <div class="img-holder"></div>
+          <div class="posts">
+            <h2>Multi-Post Stories Gain+Glory</h2>
+            <div class="flex-post">
+              <ul>
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Javascript</a></li>
+                <li><a href="#">HTML</a></li>
+              </ul>
+            </div>
+            <button type="button" class="works-button">See Project</button>
+          </div>
+        </div>
+        <div class="work-6">
+          <div class="img-holder"></div>
+          <div class="posts">
+            <h2>Multi-Post Stories Gain+Glory</h2>
+            <div class="flex-post">
+              <ul>
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Javascript</a></li>
+                <li><a href="#">HTML</a></li>
+              </ul>
+            </div>
+            <button type="button" class="works-button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <section class="about-me">
+      <div class="about-me-text">
+        <h2>About Me</h2>
```

Comment:
- [ ] Well done adding and styling about me section but please add the background following the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/159126782-66f13297-2577-465f-a095-399106261e3b.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/159126820-c2015766-36ed-4c97-a79d-7a4e11b646f1.png)